cob_collision_monitor/launch/collision_monitor.launch

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="load_robot_description" default="false"/>
   <arg name="robot" default="$(env ROBOT)"/>
   <arg name="config_path" default="$(find cob_moveit_config)/$(arg robot)/config"/>
+
+  <arg name="load_semantic_description" default="true"/>
```

Comment:
SRDF will be uploaded in bringup, so there is no real need to upload it again (per default)..
However, since the collision monitor should be started in bringup as well, the purpose of this launch file might be to start for tests on robots that aren't configured yet.
So you're right, let's keep it that way :)